### ISSUE TITLE
Incorrect textures when rendering RGB images

### ISSUE THREAD

**Issue by HelinXu** [haosulab/SAPIEN-Release#31]
**System:**
 - OS version: MacOS Big Sur 11.4
 - Python version (if applicable): Python 3.8.10
 - SAPIEN version (pip freeze | grep sapien): sapien==1.0.0rc2
 - Environment: Desktop

**Describe the bug**

I observed incorrect textures when rendering RGB images using camera.py

These textures are found to be invariant on my MacBook. Apparently, they are not something like random noise.

However, the exact same code produces different textures on another MacBook (Catalina).

Note that it happens to other instances like chairs, too.

**Screenshots**

![image](https://user-images.githubusercontent.com/59716259/126480179-1d745ed7-0b70-40ba-83db-d873ccdf879c.png)

Keyboard 13023 rendered by MacBook Big Sur 11.4

![image](https://user-images.githubusercontent.com/59716259/126479873-04a81bbd-0c6d-4d4d-b60e-1eb1b79ecace.png)

Keyboard (13023) rendered by MacBook Catalina 10.15.7

**Comment by fbxiang** [comment]
I have never observed the error pattern shown in your first picture (and even the background color does not seem right). So I really doubt there is some compatibility issue with our renderer and the graphics driver.
Since our macOS support is very experimental, and we do not have macOS developers that understand all the recent macOS versions and even new chips, I would strongly recommend using the Linux version of SAPIEN whenever possible. If you figured out potential causes (e.g. seeing similar patterns in other graphics applications) or workarounds, please let me know so I may have a try to fix it.

**Comment by HelinXu** [comment]
Thanks for the advice. I used Linux version of SAPIEN and got this:

![image](https://user-images.githubusercontent.com/59716259/126756827-e77e66cc-51dc-4181-a866-32660c83c1e2.png)

ID: 13027

Does it seem correct?

Note that the origin image looks like this

![image](https://user-images.githubusercontent.com/59716259/126757083-90c405f9-5741-47cd-a923-84b7af2ffd31.png)

It seems that the mouse beside the keyboard is missing. (This is an example from issue #29, I used nightly version as suggested in #29)
